libraries/chain/asset_evaluator.cpp

```diff
@@ -262,6 +262,8 @@ void_result asset_update_evaluator::do_evaluate(const asset_update_operation& o)
 
    if( o.new_issuer )
    {
+      FC_ASSERT( d.head_block_time() < HARDFORK_CORE_199_TIME,
```

Comment:
Be consistent with the HF check, elsewhere you use <=

Reply:
So true .. I made it `<` consistently so that I don't need to generate an 'extra block' in the unittests